examples/DotNetCore/WebJobs/WebJobHelloWorld/WebJobHelloWorld/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Configuration;
+
+namespace WebJobHelloWorld
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var hostBuilder = new HostBuilder();
+            hostBuilder.ConfigureHostConfiguration((config) =>
+            {
+                LoadConfiguration(config);
+            });
+            hostBuilder.ConfigureWebJobs(b =>
+            {
+                b.AddAzureStorageCoreServices();
+                b.AddAzureStorage();
+            });
+            hostBuilder.ConfigureLogging((context, b) =>
+            {
+                b.AddConsole();
+            });
+
+            var host = hostBuilder.Build();
+            using (host)
+            {
+                host.Run();
+            }
+        }
+
+        private static void LoadConfiguration(IConfigurationBuilder configBuilder)
+        {
+            //
+            // This application attempts to connect to Azure App Configuration to retrieve Azure Blob Storage name, and Queue Name for the Azure Web Job.
+            // It attempts to read these value from appsettings.json, environment variables and Azure App Configuration (in that order, with newer value overriding older ones).
+            configBuilder.AddJsonFile("appsettings.json")
+                .AddEnvironmentVariables();
+
+            var config = configBuilder.Build();
+            if (!string.IsNullOrEmpty(config["ConnectionString"]))
```

Comment:
Since this is an example, I am wondering if we want to fail here if the ConnectionString is not found. We can add an error message about how to provide ConnectionString via environment. Thoughts?

Reply:
Sounds good. I've changed it to throw an error if AppConfig's connection string isn't present in the environment variables or in appsettings.json.